lib/slack.ex

```diff
@@ -171,6 +171,20 @@ defmodule Slack do
   end
 
   @doc """
+  Sends `text` and `attachments` to `channel` for the given `slack` connection.
+  """
+  def send_attachments(text, attachments, channel, slack) do
+    %{
```

Comment:
Thoughts on assigning this a value like `message = %{..}`?

Then it could be `message |> JSX.encode! |> send_raw(slack)`.